TwoPaneFragments/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.0"
+    buildToolsVersion "26.0.2"
```

Comment:
Please don't update the buildToolsVersion

Reply:
Ok Mam!